syne_tune/config_space.py

```diff
@@ -1281,6 +1281,25 @@ def from_dict(d: dict) -> Domain:
     return domain
 
 
+def config_space_to_dict(
```

Comment:
Technically, we are converting to a dict so the name may a bit ambiguous (I took me some time to realize what this function was doing.

 Do you think `config_space_to_json_dict` (or `to_json_dict`) would be a better name? (and also config_space_from_dict -> `config_space_from_json_dict` or `from_json_dict`)

Reply:
Makes sense, I agree with that `to_json_dict` would be a better name.